doc/adoc/user_guide.adoc

```diff
@@ -28,7 +28,14 @@ Latest Version: {Latest_Version}
 Code available: {Repo}
 
 == Concept
-The Distribution Migration System provides an upgrade path for an installed SUSE Linux Enterprise system from one major version to another, for example, from SUSE Linux Enterprise Server 12 SP4 to SUSE Linux Enterprise Server 15. For a service pack upgrade from one Service Pack (SP) to another within a given major version release series the existing Zypper migration tool is the supported upgrade path. The distribution migration system provides the ability to upgrade across major distributions without the need to use the next major version installation media to perform the system upgrade.
+The Distribution Migration System provides an upgrade path for an installed SUSE Linux Enterprise system from one major version to another, for example, from SUSE Linux Enterprise Server 12 SP4 to SUSE Linux Enterprise Server 15. For a service pack upgrade from one Service Pack (SP) to another within a given major version release series the existing Zypper migration plugin provides the supported upgrade path. The distribution migration system provides the ability to upgrade across major distributions without the need to use the next major version installation media to perform the system upgrade.
+
+The upgrade is done via the network using the Zypper migration plugin which
+sends a request to the SUSE Customer Center (SCC), asking for an upgrade path.
+The request response contains the list of repositories required to upgrade the
+system. This requires the system to upgrade to be a SLES registered system.
+The upgrade system also supports an upgrade mode that works with non SLES
```

Comment:
s/The upgrade system /The distribution migration workflow/

s/with non SLES registered systems/with systems not registered to a repository service/